2024-03-03 16:08:44,353 CRITICAL [Errno 2] Нет такого файла или каталога: 'settings.toml'
Traceback (most recent call last):
  File "/home/maxbro/code/_Projects/PyPass/PyPass/__main__.py", line 24, in <module>
    main(argv)
  File "/home/maxbro/code/_Projects/PyPass/PyPass/__main__.py", line 17, in main
    widget = MainApp()
             ^^^^^^^^^
  File "/home/maxbro/code/_Projects/PyPass/PyPass/client/main_app/app.py", line 12, in __init__
    self.config = read_toml(FILE_SETTINGS)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/maxbro/code/_Projects/PyPass/PyPass/core/filemanage/tomlreader.py", line 9, in read_toml
    return dict(load(way))
                ^^^^^^^^^
  File "/home/maxbro/code/_Projects/PyPass/venv/lib/python3.12/site-packages/toml/decoder.py", line 133, in load
    with io.open(_getpath(f), encoding='utf-8') as ffile:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] Нет такого файла или каталога: 'settings.toml'
2024-03-04 11:02:40,376 CRITICAL encoding without a string argument
Traceback (most recent call last):
  File "/home/maxbro/code/_Projects/PyPass/PyPass/__main__.py", line 24, in <module>
    main(argv)
  File "/home/maxbro/code/_Projects/PyPass/PyPass/__main__.py", line 15, in main
    main_check()
  File "/home/maxbro/code/_Projects/PyPass/PyPass/start.py", line 7, in main_check
    create_settings()
  File "/home/maxbro/code/_Projects/PyPass/PyPass/start.py", line 12, in create_settings
    save_cryp_file_with_key(FILE_SETTINGS, SETTINGS_IN, FILE_SETTINGS_KEY)
  File "/home/maxbro/code/_Projects/PyPass/PyPass/core/cryp/files_crip.py", line 18, in save_cryp_file_with_key
    save_file_bytes(way_to_file, encrypt(data, key))
                                 ^^^^^^^^^^^^^^^^^^
  File "/home/maxbro/code/_Projects/PyPass/PyPass/core/cryp/crypo.py", line 8, in encrypt
    return Fernet(key).encrypt(bytes(data, encoding='UTF-8'))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: encoding without a string argument
2024-03-04 13:42:11,108 CRITICAL binary mode doesn't take an encoding argument
Traceback (most recent call last):
  File "/home/maxbro/code/_Projects/PyPass/PyPass/__main__.py", line 24, in <module>
    main(argv)
  File "/home/maxbro/code/_Projects/PyPass/PyPass/__main__.py", line 15, in main
    main_check()
  File "/home/maxbro/code/_Projects/PyPass/PyPass/start.py", line 7, in main_check
    create_settings()
  File "/home/maxbro/code/_Projects/PyPass/PyPass/start.py", line 12, in create_settings
    save_cryp_file_with_key(FILE_SETTINGS, write_to_toml_str(SETTINGS_IN), FILE_SETTINGS_KEY)
  File "/home/maxbro/code/_Projects/PyPass/PyPass/core/cryp/files_crip.py", line 18, in save_cryp_file_with_key
    save_file_bytes(way_to_file, encrypt(data, key))
  File "/home/maxbro/code/_Projects/PyPass/PyPass/core/filemanage/simplefiles.py", line 37, in save_file_bytes
    with open(name, 'wb', encoding='utf-8') as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: binary mode doesn't take an encoding argument
2024-03-04 13:47:02,196 DEBUG Load bytes: settings.toml
2024-03-04 13:47:02,197 CRITICAL binary mode doesn't take an encoding argument
Traceback (most recent call last):
  File "/home/maxbro/code/_Projects/PyPass/PyPass/__main__.py", line 24, in <module>
    main(argv)
  File "/home/maxbro/code/_Projects/PyPass/PyPass/__main__.py", line 15, in main
    main_check()
  File "/home/maxbro/code/_Projects/PyPass/PyPass/start.py", line 9, in main_check
    toml_type_check(SETTINGS_IN, read_toml_string(open_cryp_file_with_key(FILE_SETTINGS, FILE_SETTINGS_KEY)))
                                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/maxbro/code/_Projects/PyPass/PyPass/core/cryp/files_crip.py", line 10, in open_cryp_file_with_key
    return decrypt(load_file_bytes(way_to_file), key)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/maxbro/code/_Projects/PyPass/PyPass/core/filemanage/simplefiles.py", line 58, in load_file_bytes
    with open(name, 'rb', encoding='utf-8') as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: binary mode doesn't take an encoding argument
2024-03-04 13:47:49,549 DEBUG Load bytes: settings.toml
2024-03-04 13:47:49,593 DEBUG Load bytes: settings.toml
2024-03-04 13:51:12,733 DEBUG Load bytes: settings.toml
2024-03-04 13:51:12,777 DEBUG Load bytes: settings.toml
2024-03-04 13:51:55,719 DEBUG Load bytes: settings.toml
2024-03-04 13:51:55,762 DEBUG Load bytes: settings.toml
2024-03-04 13:51:59,164 DEBUG Load bytes: settings.toml
2024-03-04 13:51:59,207 DEBUG Load bytes: settings.toml
2024-03-04 13:52:13,595 DEBUG Load bytes: settings.toml
2024-03-04 13:52:13,645 DEBUG Load bytes: settings.toml
2024-03-04 13:54:33,880 DEBUG Load bytes: settings.toml
2024-03-04 13:54:33,923 DEBUG TOML len: 4 != 3
2024-03-04 13:54:33,924 DEBUG Load bytes: settings.toml
2024-03-04 13:54:58,970 DEBUG Load bytes: settings.toml
2024-03-04 13:54:59,013 DEBUG Load bytes: settings.toml
